src/components/layout/Layout/Layout.tsx

```diff
@@ -38,11 +41,25 @@ function Layout() {
       >
         <NavBar />
       </div>
+
+      <Progress
```

Comment:
This has a strange effect when load or reload completes, the bar animates from full all the way back down to 0. It's a bit jarring, IMO, and the bar should just disappear.